libbeat/outputs/kafka/kafka.go

```diff
@@ -270,6 +272,9 @@ func (k *kafka) newKafkaConfig() (*sarama.Config, error) {
 	}
 
 	cfg.Producer.Partitioner = k.partitioner
+
+	// TODO: figure out which metrics we want to collect
+	cfg.MetricRegistry = adapter.GetGoMetrics(monitoring.Default, "libbeat.output.kafka")
```

Comment:
yeah, this is some cleaning for a later phase. All outputs should use a similar convention in the end.